java/core/src/java/org/apache/orc/impl/DynamicByteArray.java

```diff
@@ -122,29 +121,6 @@ public int add(byte[] value, int valueOffset, int valueLength) {
     return result;
   }
 
-  /**
-   * Read the entire stream into this array.
-   * @param in the stream to read from
-   * @throws IOException
-   */
-  public void readAll(InputStream in) throws IOException {
-    int currentChunk = length / chunkSize;
-    int currentOffset = length % chunkSize;
-    grow(currentChunk);
-    int currentLength = in.read(data[currentChunk], currentOffset,
-      chunkSize - currentOffset);
-    while (currentLength > 0) {
-      length += currentLength;
-      currentOffset = length % chunkSize;
-      if (currentOffset == 0) {
-        currentChunk = length / chunkSize;
-        grow(currentChunk);
-      }
-      currentLength = in.read(data[currentChunk], currentOffset,
-        chunkSize - currentOffset);
-    }
-  }
-
```

Comment:
Sorry but I'm -1 for removing like this at 1.9.0.

Reply:
Sorry, will remove it.